cmake/protobuf-config.cmake.in

```diff
@@ -1,19 +1,30 @@
 # Version info variables
-set(PROTOBUF_VERSION        "@protobuf_VERSION@")
-set(PROTOBUF_VERSION_STRING "@protobuf_VERSION_STRING@")
+set(Protobuf_VERSION        "@protobuf_VERSION@")
+set(Protobuf_VERSION_STRING "@protobuf_VERSION_STRING@")
```

Comment:
With the addition of https://github.com/google/protobuf/pull/1624, This becomes a duplicate of the above.